samples/interactive-tutorials/events/import-user-events-gcs.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(bucketName) {
+  // [START retail_import_user_events_gcs]
+
+  // Imports the Google Cloud client library.
+  const {UserEventServiceClient} = require('@google-cloud/retail').v2;
+
+  const projectNumber = process.env['GCLOUD_PROJECT'];
+
+  const gcsBucket = `gs://${bucketName}`;
```

Comment:
same thing here, let's comment these variables and make sure developers know that they have to change them